src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -195,31 +214,63 @@ export const GettingStarted = () => {
     );
   }
 
+  const setGuideFilters = classicGuide ? (
+    <GuideFiltersClassic
+      application={application}
+      activeFilter={filter}
+      setActiveFilter={setFilter}
+      data-test-subj="onboarding--guideFilters"
+    />
+  ) : (
+    <GuideFilters
+      application={application}
+      activeFilter={filter as GuideFilterValues}
+      // @ts-ignore A/B testing type issue
```

Comment:
What's the type issue here?

Reply:
Nice! Had to fix the typing in guide filters but can now remove this ts-ignore comment. Thanks!!